scripts/publish-packages.sh

```diff
@@ -1,16 +1,37 @@
+#!/bin/sh
+
+set -eu
```

Comment:
Quick call out to [Unofficial Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) if you haven't seen it.

Reply:
When I tried adding `-o pipefail`, shellcheck told me "In POSIX sh, set option pipefail is undefined". This script could be changed to use Bash though.

Reply:
Switched to Bash in https://github.com/DataBiosphere/terra-ui/pull/4125/commits/b8bed7c6fb8945333892a19ddc7c9767ffc19fcb.